web/src/compose_ui.js

```diff
@@ -59,8 +59,7 @@ export const _get_focus_area = get_focus_area;
 export function set_focus(msg_type, opts) {
     if (window.getSelection().toString() === "" || opts.trigger !== "message click") {
         const focus_area = get_focus_area(msg_type, opts);
-        const $elt = $(focus_area);
-        $elt.trigger("focus").trigger("select");
+        $(focus_area).trigger("focus");
```

Comment:
I don't see how this would be related to the issue at hand -- isn't `opts.trigger === "message click")` in that case?

Reply:
This happens when `window.getSelection().toString()` is empty string, which makes the or-statement truthy. I checked what happens when there is a window selection, and the click still seems to clear the selection before it gets to this codeblock, which is a little odd.

But yes it is related to the issue at hand.